src/Stratis.Bitcoin.Features.PoA/Voting/VotingManager.cs

```diff
@@ -171,12 +172,17 @@ public List<VotingData> GetAndCleanScheduledVotes()
         /// <remarks>All access should be protected by <see cref="locker"/>.</remarks>
         private void CleanFinishedPollsLocked()
         {
+            bool TooOldToVoteOn(Poll poll)
+            {
+                return poll.IsPending && (this.chainIndexer.Tip.Height - poll.PollStartBlockData.Height) >= TooOldToVoteOnBlocks;
+            }
+
             // We take polls that are not pending (collected enough votes in favor) but not executed yet (maxReorg blocks
             // didn't pass since the vote that made the poll pass). We can't just take not pending polls because of the
             // following scenario: federation adds a hash or fed member or does any other revertable action, then reverts
             // the action (removes the hash) and then reapplies it again. To allow for this scenario we have to exclude
             // executed polls here.
-            List<Poll> finishedPolls = this.polls.Where(x => !x.IsPending && !x.IsExecuted).ToList();
+            List<Poll> finishedPolls = this.polls.Where(x => TooOldToVoteOn(x) || (!x.IsPending && !x.IsExecuted)).ToList();
```

Comment:
Can we add a `info` log here:
https://github.com/stratisproject/StratisFullNode/pull/709/files/0a8e2c660f39c4722b84b896215e877afa5684cd#diff-4289acc8a0560e19963016083c1bed0f2505d886b7a1f9396f1574ab0d22164eR193

To say why the vote is being excluded?

Reply:
I suggest we make it a debug log because I think it's going to pop up more than expected until such time as expiry is implemented.